website/src/styles/categories/components.scss

```diff
@@ -182,11 +182,13 @@ $transition-timing: 300ms;
   margin-top: 0.8rem;
 }
 
-.ds-component__buttons-wrapper {
+.ds-component__buttons-wrapper,
+.ds-component__checkboxes-wrapper {
```

Comment:
should we make this a more general selector? or are we going to add all components here?